test/jdk/java/awt/regtesthelpers/PassFailJFrame.java

```diff
@@ -149,6 +149,15 @@
  * <p>
  * Before returning from {@code awaitAndCheck}, the framework disposes of
  * all the windows and frames.
+ * <p id="forcePassAndFail">
+ * There are two other static methods available,
+ * {@link #forcePass() forcePass} and {@link #forceFail() forceFail}, which can
+ * be used in semi-automated tests to indicate that the test can be forcefully
+ * Passed/Failed. These methods are not immediately evaluated, but it decrements
+ * a counter and return. The evaluation happens in {@code awaitAndCheck} and
+ * the test will be forcefully Passed/Failed accordingly.
+ * Code examples are given in the corresponding method javadoc.
+ * <p>
```

Comment:
```suggestion
 *
 * <p id="forcePassAndFail">
 * For semi-automatic tests, use {@code forcePass} or
 * {@code forceFail} methods to forcibly pass or fail the test
 * when it's determined that the required conditions are already met
 * or cannot be met correspondingly.
 * These methods release {@code awaitAndCheck}, and
 * the test will complete successfully or fail.
 * <p>
 * Refer to examples of using these methods in the description of the
 * {@link #forcePass() forcePass} and {@link #forceFail() forceFail} methods.
```

The empty `<p>` in the end before a heading is redundant and will be ignored.